azkaban-db/src/main/sql/create.containerization-tables-all.sql

```diff
@@ -1,71 +1,104 @@
+-- Definition for image_types table. This table is used for storing different image types
 CREATE TABLE IF NOT EXISTS image_types (
-  id               INT         NOT NULL PRIMARY KEY AUTO_INCREMENT,
-  type             VARCHAR(64) NOT NULL UNIQUE,
+  id               INT             NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  name             VARCHAR(64)     NOT NULL UNIQUE,
   description      VARCHAR(2048),
   active           BOOLEAN,
   deployable       VARCHAR(64),
   created_on       TIMESTAMP       NOT NULL DEFAULT CURRENT_TIMESTAMP,
   created_by       VARCHAR(64)     NOT NULL,
-  modified_on      TIMESTAMP       DEFAULT NULL,
-  modified_by      VARCHAR(64)
+  modified_on      TIMESTAMP       NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
Add set it to current_timestamp on update as well

Reply:
I tried adding on update clause earlier, but build fails due to the existing custom code which tries to execute all db scripts. DatabaseSetup which invokes the custom code to execute the db scripts fails. So that piece of code is unable to recognize theon update clause. For now, we can update the update timestamp from the code. This can be taken up separately while implementing DB schema syncing and auto versioning task.